WiserTaskScheduler/WiserTaskScheduler/Modules/Wiser/Services/WiserService.cs

```diff
@@ -147,31 +147,40 @@ public async Task<List<TemplateSettingsModel>> RequestConfigurations()
-                request.Headers.Add("Authorization", $"Bearer {AccessToken}");
-
+                var configurationPaths = String.IsNullOrWhiteSpace(wiserSettings.ConfigurationPath) ? new[] { "" } : wiserSettings.ConfigurationPath.Split(";");
                 using var client = new HttpClient();
-                try
+                
+                var configurations = new List<TemplateSettingsModel>();
+                
+                foreach (var configurationPath in configurationPaths)
                 {
-                    var response = client.Send(request);
+                    try
+                    {
+                        var request = new HttpRequestMessage(HttpMethod.Get, $"{wiserSettings.WiserApiUrl}api/v3/templates/entire-tree-view?startFrom=SERVICES{(String.IsNullOrWhiteSpace(configurationPath) ? "" : $",{configurationPath}")}&environment={environment}");
+                        request.Headers.Add("Authorization", $"Bearer {AccessToken}");
+
+                        var response = client.Send(request);
+
+                        if (response.StatusCode != HttpStatusCode.OK)
+                        {
+                            logService.LogCritical(logger, LogScopes.RunStartAndStop, logSettings, "Failed to get configurations from the Wiser API.", "WiserService");
+                            return null;
+                        }
+
+                        using var reader = new StreamReader(response.Content.ReadAsStream());
+                        var body = reader.ReadToEnd();
+                        var templateTrees = JsonConvert.DeserializeObject<List<TemplateTreeViewModel>>(body);
 
-                    if (response.StatusCode != HttpStatusCode.OK)
+                        configurations.AddRange(FlattenTree(templateTrees));
+                    }
+                    catch (Exception e)
                     {
-                        logService.LogCritical(logger, LogScopes.RunStartAndStop, logSettings, "Failed to get configurations from the Wiser API.", "WiserService");
+                        logService.LogCritical(logger, LogScopes.RunStartAndStop, logSettings, $"Failed to get configurations from the Wiser API.\n{e.Message}\n{e.StackTrace}", "WiserService");
                         return null;
                     }
-
-                    using var reader = new StreamReader(response.Content.ReadAsStream());
-                    var body = reader.ReadToEnd();
-                    var templateTrees = JsonConvert.DeserializeObject<List<TemplateTreeViewModel>>(body);
-
-                    return FlattenTree(templateTrees);
-                }
-                catch (Exception e)
-                {
-                    logService.LogCritical(logger, LogScopes.RunStartAndStop, logSettings, $"Failed to get configurations from the Wiser API.\n{e.Message}\n{e.StackTrace}", "WiserService");
```

Comment:
Het is misschien beter om de hele exception in de log te zetten i.p.v. alleen de message en stack trace zodat ook alle inner exceptions worden weggeschreven naar de log. Bijvoorbeeld zo:
```c#
logService.LogCritical(logger, LogScopes.RunStartAndStop, logSettings, $"Failed to get configurations from the Wiser API.\n{e}", "WiserService");
```